nativelib/src/main/scala/scala/scalanative/native/time.scala

```diff
@@ -0,0 +1,84 @@
+package scala.scalanative
+package native
+
+import native.signal.sigevent
+import posix.sys.types.{timer_t, pid_t, clockid_t}
+
+// http://man7.org/linux/man-pages/man7/time.7.html
```

Comment:
Here we don't aim to do anything linux-specific. Our goal is to do purely bindings for documented POSIX APIs. e.g. http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/time.h.html